installer/conf/container.conf

```diff
@@ -31,6 +31,30 @@
   log_level info
 </filter>
 
+<source>
+  type dockerhealth
```

Comment:
are we sure that we are releasing this as part of next agent release ? if not, we should not merge this until we know ux is ready and all works/make sense.

Reply:
huh i see that its a standup PR and took changes from ci_feature into your branch. makes sense now :)